shared/types/proto/events.proto

```diff
@@ -1,15 +1,14 @@
 syntax = "proto3";
-
 import "google/protobuf/any.proto";
 
 package pocket;
 
-option go_package = "/types/";
+option go_package = "github.com/pokt-network/pocket/shared/types;types";
```

Comment:
This has allowed me to use the shared proto definitions in P2P's definitions (_I needed that_). But since this is not part of this PR, I think I will remove this change until It's shipped with what it makes most sense with.